pymeasure/instruments/thurlby/thurlby1905a.py

```diff
@@ -0,0 +1,126 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2025 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+from pymeasure.errors import Error, RangeError
+from pymeasure.instruments import Instrument
+
+
+class Thurlby1905a(Instrument):
+    """Represents the Thurlby 1905a intelligent digital multimeter
+
+    This instrument only provides measurement output via an RS-232 serial port.
+    The serial port can only be read from, not written to.
+
+    Earlier models had a Baud rate of 2400, later ones 9600.
+
+    .. code-block python
+
+        from pymeasure.instruments.thurlby import Thurlby1905a
+
+        dmm = Thurlby1905a("ASRL/dev/ttyACM0::INSTR")
+        output = dmm.measurement
+
+    """
+
+    def __init__(self, adapter, name="Thurlby 1905a", **kwargs):
+        super().__init__(
+            adapter,
+            name,
+            includeSCPI=False,
+            read_termination="\r",
+            **kwargs,
+        )
```

Comment:
As different devices could have different baud rates, I'd suggest to add it to the signature:

```suggestion
    def __init__(self, adapter, name="Thurlby 1905a", baud_rate=2400, **kwargs):
        super().__init__(
            adapter,
            name,
            includeSCPI=False,
            read_termination="\r",
            baud_rate=baud_rate,
            **kwargs,
        )
```

Change the default to 9600, if you prefer.